tests/BaseTestCase.php

```diff
@@ -9,28 +9,4 @@
  */
 class BaseTestCase extends TestCase
```

Comment:
I think we can completely drop this class, it was only here to keep BC with older phpunit versions

Reply:
Sure